Kickstarter-iOS/Tests/ViewModels/AppDelegateViewModelTests.swift

```diff
@@ -308,30 +308,32 @@ final class AppDelegateViewModelTests: TestCase {
     XCTAssertFalse(self.facebookAppDelegate.didFinishLaunching)
     XCTAssertFalse(self.facebookAppDelegate.openedUrl)
 
-    self.vm.inputs.applicationDidFinishLaunching(application: UIApplication.sharedApplication(),
+    self.vm.inputs.applicationDidFinishLaunching(application: UIApplication.shared,
                                                  launchOptions: [:])
 
     XCTAssertTrue(self.facebookAppDelegate.didFinishLaunching)
     XCTAssertFalse(self.facebookAppDelegate.openedUrl)
 
-    self.vm.inputs.applicationOpenUrl(application: UIApplication.sharedApplication(),
-                                      url: NSURL(string: "http://www.fb.com")!,
-                                      sourceApplication: nil,
-                                      annotation: 1)
+    let result = self.vm.inputs.applicationOpenUrl(application: UIApplication.shared,
+                                                   url: URL(string: "http://www.fb.com")!,
+                                                   sourceApplication: nil,
+                                                   annotation: 1)
+    XCTAssertFalse(result)
```

Comment:
there were a bunch of unused return value warnings with calling `applicationOpenUrl` so now im asserting on those values